pkg/image/client.go

```diff
@@ -47,7 +47,7 @@ func New(c Config) (*Client, error) {
 	return client, nil
 }
 
-func (i *Client) RemoveImage(ctx context.Context, image string) error {
+func (i *Client) RemoveRelease(ctx context.Context, image string) error {
```

Comment:
The name of this method suggests something different 😄 Maybe better to use `RemoveReleaseFromNodeImageStatus` or something like that